backend/src/main/conf/local.json

```diff
@@ -15,5 +31,25 @@
         "window.tolerance.secs": 60
       }
     }
+  },
+  "curatorOptions": {
+    "connection.url": "",
+    "namespace": "fkprof",
+    "connection.timeout.ms": 5000,
+    "session.timeout.ms": 5000,
```

Comment:
Curator has it with 60 seconds. I was doing some local testing where I was killing the leader and bringing it up again, so reduced session timeout. Will keep such test-related changes to config.json in test directory